Globus transfer handling fails to initialize

### Step to reproduce

1. Register Globus resource
2. Create a Tale, add globus resources to dataSet
3. Launch a Tale
4. `cat ../data/<globus_resource>`

### Expected outcome
See the content of the file

### Actual outcome
Despair and tracebacks:

```
[2021-05-02 18:38:29,560] ERROR: Traceback (most recent call last):
[2021-05-02 18:38:29,561] ERROR:   File "/girder/plugins/wt_data_manager/server/lib/transfer_manager.py", line 24, in run
[2021-05-02 18:38:29,561] ERROR:     self.transferHandler.run()
[2021-05-02 18:38:29,561] ERROR:   File "/girder/plugins/wt_data_manager/server/lib/tm_utils.py", line 34, in run
[2021-05-02 18:38:29,561] ERROR:     self.transfer()
[2021-05-02 18:38:29,562] ERROR:   File "/girder/plugins/wt_data_manager/server/lib/handlers/globus.py", line 30, in transfer
[2021-05-02 18:38:29,562] ERROR:     self._maybeStartServer()
[2021-05-02 18:38:29,562] ERROR:   File "/girder/plugins/wt_data_manager/server/lib/handlers/globus.py", line 106, in _maybeStartServer
[2021-05-02 18:38:29,562] ERROR:     self.server.start()
[2021-05-02 18:38:29,563] ERROR:   File "/girder/plugins/globus_handler/server/server.py", line 57, in start
[2021-05-02 18:38:29,563] ERROR:     (self.endpointId, self.endpointName) = self._createEndpoint()
[2021-05-02 18:38:29,563] ERROR:   File "/girder/plugins/globus_handler/server/server.py", line 223, in _createEndpoint
[2021-05-02 18:38:29,563] ERROR:     _runGCCommand('%s/globusconnectpersonal' % self.gcDir, '-setup', setupKey, '-dir',
[2021-05-02 18:38:29,563] ERROR:   File "/girder/plugins/globus_handler/server/server.py", line 19, in _runGCCommand
[2021-05-02 18:38:29,564] ERROR:     raise Exception(msg)
[2021-05-02 18:38:29,564] ERROR: Exception: Command ('/opt/globusconnectpersonal/globusconnectpersonal', '-setup', '...', '-dir', '/home/girder/.WholeTale/0baa7c04-1ff6-4cf5-82b3-00dd47e828d9') failed with exit code 2:
[2021-05-02 18:38:29,564] ERROR: b'usage: register [-h] [--attributes [file:PATH|JSON_DATA]] [-n NAME]\n                [--description DESCRIPTION] [--owner OWNER] [-z]\n 
```

### NOTE
Switching order of args from `--setup ... --dir ...` to `--dir ... --setup ...` fixes the issue. Globuspersonalconnect passes everything that comes after --setup to separate routine called `register`.
Update: then I get a different error:

```
Exception: Command ('/opt/globusconnectpersonal/globusconnectpersonal', '-debug', '-dir', '/home/girder/.WholeTale/0baa7c04-1ff6-4cf5-82b3-00dd47e828d9', '-setup', 'e916cf72-97a0-4e49-9f03-3019ae33b3b0') failed with exit code 1: 
b"Could not read the directory '/home/girder/.WholeTale/0baa7c04-1ff6-4cf5-82b3-00dd47e828d9' with configuration files.\nCreating the directory... Done\nrelaytool setup exit with status 1, indicating failure. use debug mode for full output\nrelaytool failed, full error info available in register.log\n\nSetup did not complete successfully.\nYou may want to check /home/girder/.WholeTale/0baa7c04-1ff6-4cf5-82b3-00dd47e828d9/lta/register.log for more information\n"

$ cat register.log 
[INFO] cli.py::190 [main] got --setup-key, skip endpoint creation...
[INFO] cli.py::196 [main] starting relaytool setup
[WARNING] relaytool.py::136 [invoke_relaytool] relaytool setup exit with status 1, indicating failure. use debug mode for full output
[INFO] cli.py::200 [main] relaytool stdout:
-----
('',)-----
[INFO] cli.py::201 [main] relaytool stderr:
-----
('',)
-----
[INFO] cli.py::202 [main] full trace of relaytool failure
Traceback (most recent call last):
  File "lib/cli.py", line 198, in main
  File "lib/relaytool.py", line 151, in run_relaytool_setup
  File "lib/relaytool.py", line 146, in invoke_relaytool
lib.relaytool.RelayToolFailureError: ('relaytool setup failed', CompletedProcess(args='/opt/globusconnectpersonal/gt_amd64/bin/relaytool', returncode=1, stdout=b'', stderr=b''))
[ERROR] cli.py::203 [main] relaytool failed, full error info available in register.log
```
```

--- a/Dockerfile.py3
+++ b/Dockerfile.py3
@@ -74,9 +74,9 @@ RUN set -ex \
 	&& ldconfig
 
 # install GCP client
-ENV GCP_URL=https://downloads.globus.org/globus-connect-personal/linux/stable/globusconnectpersonal-latest.tgz
-RUN wget -qO- $GCP_URL | tar xz -C /opt && \
-  mv /opt/globusconnectpersonal-* /opt/globusconnectpersonal
+# ENV GCP_URL=https://downloads.globus.org/globus-connect-personal/linux/stable/globusconnectpersonal-latest.tgz
+ENV GCP_URL=https://github.com/whole-tale/globus_handler/releases/download/gcp-3.0.4/globusconnectpersonal-3.0.4.tar.gz
+RUN wget -qO- $GCP_URL | tar xz -C /opt 
 
 RUN groupadd -r girder \
   && useradd --no-log-init -s /bin/bash -p $(openssl rand -base64 32) -m -r -g girder girder \

